media/css/firefox/welcome13.scss

```diff
@@ -183,13 +183,17 @@ $image-path: '/media/protocol/img';
 
         .mzp-c-footer-links-social li {
             a.twitter {
-                background-image: url('/media/protocol/img/icons/social/twitter/white.svg');
+                background-image: url('/media/protocol/img/icons/social/x/white.svg');
             }
 
             a.instagram {
                 background-image: url('/media/protocol/img/icons/social/instagram/white.svg');
             }
 
+            a.mastodon {
+                background-image: url('/media/img/logos/social/mastodon-white.svg');
```

Comment:
**nitpick:** I noticed the Mastodon icons were black in some of the Welcome pages, so I updated them with this commit.